src/components/expansion-panel/expansion-panel.ts

```diff
@@ -0,0 +1,239 @@
+import { LitElement, html } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { Constructor } from '../common/mixins/constructor';
+import { EventEmitterMixin } from '../common/mixins/event-emitter';
```

Comment:
```suggestion
import { Constructor } from '../common/mixins/constructor.js';
import { EventEmitterMixin } from '../common/mixins/event-emitter.js';
```
Once TypeScript 4.7 hits official, we will try to enforce ESM rules for the package. But currently this will break under Webpack 5